client/js/widgets/widget.js

```diff
@@ -535,6 +535,9 @@ export class Widget extends StateManaged {
           case 'unshift':
             v[o](x);
             break;
+          case 'in':
+            v = y.indexOf(x) != -1;
+            break;
```

Comment:
User may want to use the actual `in` function from JavaScript which allows to identify whether a certain property exists in an object, so we should check whether this is an array or not.

```suggestion
          case 'in':
            v = (Array.isArray(y)) ? y.indexOf(x) != -1 : x in y
            break;
```